tests/testthat/test-ui-layout.R

```diff
@@ -51,13 +51,12 @@ test_that("cXlayout12", {
     check_ui_test(cXlayout12())
 
     warning('Fewer pie charts displayed than expected, does not match web example')
-    warning("Colors for drug are different, two pieces are green, but one should be orange")
```

Comment:
This is not fixed in the latest version.  Warning should not be removed.
<img width="206" alt="Screen Shot 2019-12-16 at 1 31 58 PM" src="https://user-images.githubusercontent.com/9085025/70944744-8ae03c80-2008-11ea-90ae-17f148f6fc0a.png">